### ISSUE TITLE
Experimental branch

### ISSUE THREAD

**Issue by jnvsor** [kint-php/kint#195]
Currently looking at my outstanding PRs alone it's impossible to merge them all because they have merge conflicts.

This makes testing them even harder since you can't have a build that tests them all at once. Add to this there are a lot of users that would rather have features _now_ than stability and I'd like to suggest an experimental branch.
- No guarantees against force push
- PRs get in early for testing
- Good patches get cherry picked into master
- More people can help @raveren test
- Add a section to the bottom of README.md to explain it

Basically a sandbox to speed up and distribute testing.

(Alternatively, you could just throw up a new tag for the first time in 2 years and use master a bit more liberally. Reverts aren't the end of the world :)

**Comment by jnvsor** [comment]
Going to go ahead and bump this, I have a fairly large experimental branch of my own right now: https://github.com/jnvsor/kint/tree/experimental

Additionally, you might want to consider a branch for the next major version to incorporate breaking changes (I have a few PRs in mind that fix bugs/improve things in ways that could technically break behaviour in some rare cases)

**Comment by raveren** [comment]
all further Kint development discussion is now in #208 I believe
